src/msw/webview_ie.cpp

```diff
@@ -301,8 +301,62 @@ void wxWebViewIE::SetZoom(wxWebViewZoom zoom)
 
 void wxWebViewIE::SetZoomFactor(float zoom)
 {
-    SetIEOpticalZoom(zoom);
-    SetIETextZoom(zoom);
+    VARIANT zoomVariant;
```

Comment:
Sorry, but I don't understand what's going on here: this function now seems to duplicate most (but not all) of `SetIEXXXZoom()` functions and this really doesn't seem good. Why can't we reuse them any more? We should refactor the code to have `SetIEXXXZoomFactor(float)` and call them from both places, i.e. the existing `SetIEXXXZoom()`, after translating `wxWebViewZoom` to `float`, and from the new `SetZoomFactor()`.